### ISSUE TITLE
Alarm randomly sets to home

### ISSUE THREAD

**Issue by danflynndesign** [nielsfaber/alarmo#835]
### Alarmo Version

1.9.13

### HA Version

2023.11

### Bug description

About twice every 24 hours, the alarm randomly sets to home. There is no automation that sets it, but checking the log shows that alarmo set the status to armed home. This has been an issue since installing alarmo on a fresh install of home assistant, where everything was set up from scratch and not a backup.

### Steps to reproduce

Install alarmo.
I do have an automation that has "Alarm set to..." spoken on my nest hub displays whenever it is set to home or away or disarmed.
And finally, just wait, and the most recents were that my alarm was set to home at 12:27 am and 2:04pm in the past 24 hours. The home assistant app was not opened and no actions were performed.

### Relevant log output

```shell
Downstairs Display changed to Playing triggered by automation Alarm | Armed Home Notification triggered by state of Alarmo 2:03:58 PM - 1 hour ago 

Downstairs Display changed to ldle triggered by automation Alarm | Armed Home Notification triggered by state of Alarmo 2:03:56 PM -1 hour ago

Alarm | Armed Home Notification triggered by state of Alarmo changed to Armed home 2:03:54 PM -1 hour ago - Traces 

Alarmo changed to Armed home 2:03:54 PM -1 hour ago
```

**Comment by nielsfaber** [comment]
Alarmo does not arm on its own.
Did you configure MQTT for alarmo?
If so, it’s likely the cause.

**Comment by danflynndesign** [comment]
Hi Nielsfaber, thanks for the quick relply! I have discovered this issue and this is now resolved. Closing this ticket.
I have not configured MQTT. I read on some forums that others had the same issue recently so thought it was a bug. Instead a sensor was falsely triggering the alarm, and as this was a fresh install of HA, I didn't have any audible notification when it was triggered. Instead, after 30 minutes of it being triggered, it would return to its original state. That's what was "randomly" causng the 'alarm disarmed' or 'alarm armed home' notifications. Take care!
